src/secrets/CommandEnv.ts

```diff
@@ -175,14 +192,26 @@ class CommandEnv extends CommandPolykey {
           // Here we want to switch between the different usages
           const platform = os.platform();
           if (cmd != null) {
-            // If a cmd is| provided then we default to exec it
+            // If a cmd is provided then we default to exec it
             switch (platform) {
               case 'linux':
-              // Fallthrough
```

Comment:
Oh. I just did it for clarity, as in other places, the fallthrough comment has been omitted. Especially in error composition in Polykey, Polykey-CLI, and js-rpc. I was just being consistent with that. I can change this back if needed, but how do I handle the other switch statements with that?

Reply:
Add it back in. If you see it missing in other places then add it in there. I thought the linter complains about this but it might just be my IDE.

Reply:
Yeah, I think your IDE does some extra processing over the regular standards we have. Can you take a look at `Polykey:src/network/utils.ts:487` and see if your IDE still complains for that? If so, then there are a couple places where we would need to add the fallthrough comment.

Reply:
Seems my IDE doesn't complain about that. All the same messing up a fallthrough in a switch case is a common bug. and annoying to debug because intention can be unclear. Its best to be explicit about the fallthough.

If you're worried about readability then just put the comment on the same line
```ts
switch (thing) {
// ...
    case '"': // Fallthrough
    case "'": // Fallthrough
    case '`': // Fallthrough
// ...
}
```

Reply:
Okay, yeah, this looks clean and is also informative. Ill make this change across all relevant cases in this repo.